src/pages/Browse/index.spec.tsx

```diff
@@ -252,24 +266,31 @@ describe(__filename, () => {
     onSelectFile(path);
 
     expect(dispatch).toHaveBeenCalledWith(
-      versionActions.updateSelectedPath({
+      versionsActions.updateSelectedPath({
         versionId: version.id,
         selectedPath: path,
       }),
     );
+
+    expect(history.push).toHaveBeenCalledWith(
```

Comment:
Because you've added this assertion to the test, you should probably either update the description of the test, or have a separate test with just this assertion. As it stands, this test is just called "dispatches updateSelectedPath() when a file is selected".